.ci/Vagrantfile.fedora33

```diff
@@ -20,8 +20,7 @@ Vagrant.configure("2") do |config|
       cat << EOF | dnf -y shell && break
 config exclude kernel,kernel-core
```

Comment:
Wondering if this line was related to the `update` line you removed (i.e.; "update, but don't install kernel updates")?

Reply:
Yes it is, but I'd like to keep this line, which basically says "no matter what we do below, exclude kernel from it".

Some Fedora version needed update in the past, and this need might re-surface, so there's a chance someone will add `update` later, and they will definitely forget about excluding the kernel, which is a big package for which there's almost always an update available.

So yes, the line does not make sense per se currently, but it might come handy later, and since it does not add any overhead or something, I'd rather keep it.

Alternatively, we can re-add `update` and comment it out, together with the exclude. Let me know if you prefer it this way.

Reply:
(given that the comments are available in that context -- I'm not entirely sure)